holmes/plugins/sources/jira/__init__.py

```diff
@@ -36,18 +36,51 @@ def fetch_issues(self) -> List[Issue]:
         except requests.RequestException as e:
             raise ConnectionError("Failed to fetch data from Jira.") from e
 
-    def convert_to_issue(self, jira_issue):
+    def convert_to_issue(self, jira_issue, description: Optional[str] = None):
+        description = self.extract_description(jira_issue)
         return Issue(
             id=jira_issue["id"],
             name=jira_issue["fields"]["summary"],
             source_type="jira",
             source_instance_id=self.url,
             url=f"{self.url}/browse/{jira_issue['key']}",
+            description=description,
             raw=jira_issue,
         )
 
         # status=jira_issue["fields"]["status"]["name"],
-        # description=jira_issue.get("fields", {}).get("description", ""),
+
+    def extract_description(self, jira_issue) -> str:
+        """
+        Extracts and formats the issue description.
+        """
+        description_blocks = (
```

Comment:
Is this logic the same for both Jira and JSM?

Reply:
yea